.github/workflows/release.yml

```diff
@@ -1,48 +1,42 @@
 name: Release
 
-# Temporarily disable until issues resolved
 on:
-#   release:
-#     types: [published]
+  release:
+    types: [published]
```

Comment:
Q: Should it stay like that, or should we create other convention for creating releases?